packages/frontend/app/lib/message.server.ts

```diff
@@ -20,10 +21,9 @@ export const messageStorage = createCookieSessionStorage<
     sameSite: 'lax',
     maxAge: ONE_MINUTE_IN_S,
     secrets: ['MY_SUPER_SECRET_TOKEN'],
-    secure:
-      process.env.COOKIE_SECURE === undefined
-        ? true
-        : ['true', 't', '1'].includes(process.env.COOKIE_SECURE.toLowerCase())
+    secure: process.env.ENABLE_INSECURE_MESSAGE_COOKIE
+      ? !parseBool(process.env.ENABLE_INSECURE_MESSAGE_COOKIE)
+      : process.env.NODE_ENV === 'production'
```

Comment:
basically, you can pass in `ENABLE_INSECURE_MESSAGE_COOKIE=true` to set this to false otherwise it uses `NODE_ENV` like we currently do.

this means it's secure by default and when running in dev mode we dont need to set `ENABLE_INSECURE_MESSAGE_COOKIE`

for good measure here's a state table:

| ENABLE_INSECURE_MESSAGE_COOKIE | NODE_ENV      | Secure |
| ------------------------------ | ------------- | ------ |
| true                           | any           | false  |
| false                          | any           | true   |
| undefined                      | 'production'  | true   |
| undefined                      | 'development' | false  |
| undefined                      | undefined     | true   |